mne/channels/channels.py

```diff
@@ -1536,3 +1539,150 @@ def make_1020_channel_selections(info, midline="z"):
                       for selection, picks in selections.items()}
 
     return selections
+
+
+def combine_channels(instance, groups, method='mean', keep_stim=False,
+                     drop_bad=False):
+    """Combine channels based on regions of interest (ROIs).
```

Comment:
You use the terms `groups` further down and only use the acronym ROI in the example below. Combinations are also not specified based on regions, but lists of channels/indices. So the current wording seems more about how you would use the function rather than what it does. Perhaps..

```suggestion
    """Combine channels based on specified channel grouping.
```